unit-tests/src/test/scala/scala/scalanative/posix/TimeTest.scala

```diff
@@ -133,6 +185,61 @@ class TimeTest {
     }
   }
 
+  @Test def strptimeDoesNotWriteMemoryOutsideStructTm(): Unit = {
+    Zone { implicit z =>
+      // Linux 56 Bytes, Posix specifies 36 but allows more.
+      val tmBufSize = 7
+      val tmBuf     = alloc[Ptr[Byte]](tmBufSize) // will zero/clear all bytes
+      val tmPtr     = tmBuf.asInstanceOf[Ptr[tm]]
+
+      val cp =
+        strptime(c"Fri Mar 31 14:47:44 EDT 2017", c"%a %b %d %T %Z %Y", tmPtr)
+
+      assertNotNull(s"strptime() returned unexpected null pointer", cp)
+
+      val ch = cp(0)
+      assertEquals("strptime() result is not NUL terminated", ch, '\u0000')
+
+      val tm_gmtoff = tmBuf(5) // tm_gmtoff is outside posix minimal range.
```

Comment:
You never use `™_gmtoff`

Reply:
You are correct. That assertion and the one after it were totally useless. What was I thinking?  

The two assertions are now meaningful in the current evolution.